app/templates/pricing.hbs

```diff
@@ -1,4 +1,10 @@
+<div class="sixteen wide column lead {{if this.smallLead 'small'}}">
+  <Widgets::SafeImage class="background" @src="https://user-images.githubusercontent.com/56407566/92262388-a7a30400-eef8-11ea-9a63-8cf78ce4dfe0.png" />
```

Comment:
??

Reply:
@mariobehling has said to add a header image in issue description, so I added this one.

Reply:
This should be in assets, not uploaded to github